--- bootchk-resynthesizer-e637010/src/Makefile.am.linking	2012-03-30 10:43:38.058526390 +0200
+++ bootchk-resynthesizer-e637010/src/Makefile.am	2012-03-30 10:43:45.808525046 +0200
@@ -81,7 +81,7 @@ INCLUDES =\
 # Make explicit dependency on libm for atan2()
 # Fixes change to linker ld on some platforms that causes libm not to be found through other libraries.
 # resynthesizer-gui does not actually depend on libm
-LDADD = $(GIMP_LIBS) libimagesynth.a -lm
+LDADD = $(GIMP_LIBS) libimagesynth.a -lm -lgthread-2.0
 
 
 
